reference/json-rpc/filecoin.md

```diff
@@ -2,14 +2,41 @@
 
 ## FilecoinAddressToEthAddress
```

Comment:
Why is this API in `filecoin.md`? I am not able to see it on docs UI. It is a API in `eth.go` - code is [here](https://github.com/filecoin-project/lotus/blob/49b57971b35d5a161eacc8988a3c6016b52fb5af/node/impl/full/eth.go#L212)

Reply:
I'll delete this and migrate to eth.md